backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidLiveWallpaperService.java

```diff
@@ -568,6 +568,51 @@ public void run () {
 			}
 		}
 
+		// zoom from last onZoomChanged
+		boolean zoomConsumed = true;
+		float zoom = 0.0f;
+
+		@Override
+		public void onZoomChanged (float zoom) {
+
+			// it spawns too frequent on some devices - its annoying!
+			// if (DEBUG)
+			// Log.d(TAG, " > AndroidWallpaperEngine - onZoomChanged(" + zoom + ") " + hashCode()
+			// + ", linkedApp: " + (linkedApp != null));
+
+			this.zoomConsumed = false;
+			this.zoom = zoom;
+
+			// can fail if linkedApp == null, so we repeat it in Engine.onResume
```

Comment:
This comment is either outdated or wrong as this isn't repeated in Engine.onResume?

Reply:
I indeed forgot some calls of the new method, now it should be all right!